.github/workflows/gradle.yml

```diff
@@ -0,0 +1,36 @@
+name: Gradle Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+
+      - name: Run Gradle Build
+        run: ./gradlew build
+        working-directory: TrafficCapture
+
+      - name: Run Tests with Coverage
+        run: ./gradlew test jacocoTestReport
+        working-directory: TrafficCapture
+
+      - name: Upload to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          files: TrafficCapture/captureKafkaOffloader/build/reports/jacoco/test/jacocoTestReport.xml,
```

Comment:
Do you know if its possible to use any pattern matching/regex here to have this as a single line so that it's easier to maintain in future

Reply:
I know that it is possible to use wildcards, but not in the way I had hoped.
e.g `TrafficCapture/*/build/reports/jacoco/test/jacocoTestReport.xml` will NOT lead including all the files in our case.
One of the cases that would work is if, for example we had jacocoTestReport1, jacocoTestReport2, jacocoTestReport3 - then all these would be included if `full-path-here/jacocoTestReport*.xml` was used instead of each file respectively.